src/ConnectedMachine/docs/Update-AzConnectedMachine.md

```diff
@@ -81,8 +84,54 @@ Update a machine to clean a field
 
 ## PARAMETERS
 
+### -AgentUpgradeCorrelationId
+The correlation ID passed in from RSM per upgrade.
+
+```yaml
+Type: System.String
+Parameter Sets: UpdateExpanded, UpdateViaIdentityExpanded
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -AgentUpgradeDesiredVersion
+Specifies the version info w.r.t AgentUpgrade for the machine.
+
+```yaml
+Type: System.String
+Parameter Sets: UpdateExpanded, UpdateViaIdentityExpanded
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -AgentUpgradeEnableAutomaticUpgrade
```

Comment:
better to rename it to "AgentUpgradeEnableAutomatic"

Reply:
changed in the latest commit